intl.emu

```diff
@@ -0,0 +1,351 @@
+          1. Let _int_ be 1.
+        1. If _m_ contains the code unit 0x002E (FULL STOP) and _maxPrecision_ &gt; _minPrecision_, then
+          1. Let _cut_ be _maxPrecision_ - _minPrecision_.
+          1. Repeat, while _cut_ &gt; 0 and the last code unit of _m_ is 0x0030 (DIGIT ZERO),
+            1. Remove the last code unit from _m_.
+            1. Set _cut_ to _cut_ - 1.
+          1. If the last code unit of _m_ is 0x002E (FULL STOP), then
+            1. Remove the last code unit from _m_.
+        1. Return the Record { [[FormattedString]]: _m_, [[RoundedNumber]]: _xFinal_, [[IntegerDigitsCount]]: _int_, [[RoundingMagnitude]]: _e_–_p_+1 }.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-torawfixed" type="abstract operation">
+      <h1>
+        ToRawFixed (
+          _x_: <ins>either a </ins> non-negative mathematical value<ins> or a non-NaN, finite, non-negative Decimal128 object</ins>,
+          _minFraction_: an integer in the inclusive interval from 0 to 100,
+          _maxFraction_: an integer in the inclusive interval from 0 to 100,
+          _roundingIncrement_: an integer,
+          _unsignedRoundingMode_: a specification type from the Unsigned Rounding Mode column of <emu-xref href="#table-intl-unsigned-rounding-modes"></emu-xref>, or *undefined*,
+        ): a Record with fields [[FormattedString]] (a String), [[RoundedNumber]] (a mathematical value), [[IntegerDigitsCount]] (an integer), and [[RoundingMagnitude]] (an integer)
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>
+          <p>
+            <ins>For mathematical values, it</ins> <del>It</del> involves solving the following equation, which returns a valid mathematical value given integer inputs:
+          </p>
+          <emu-eqn id="eqn-ToRawFixedFn" aoid="ToRawFixedFn">
+            ToRawFixedFn(_n_, _f_) = _n_ × 10<sup>–_f_</sup>
+          </emu-eqn>
+	  <p><ins>For Decimal128 object arguments, it involves working with the serialized decimal string representation of the argument.</ins></p>
+	  <p><ins>In what follows, when we say <q>_a_ ≤ _b_</q>, where _a_ is a Decimal128 object and _b_ is an integer, we understand this to mean Call(_x_, *cmp*, Construct(Decimal128, Call(_r1_.[[toString]])) in « *-1*<sub>𝔽</sub>, *+0*<sub>𝔽</sub> ». Analogously, <q>_a_ ≥ _b_</q>, where _a_ is a Decimal128 object and _b_ is an integer, we understand this to mean Call(_x_, *cmp*, « Construct(Decimal128, Call(_r1_, *toString*)) » in « *+0*<sub>𝔽</sub>, *1*<sub>𝔽</sub> ».</ins></p>
+        </dd>
+      </dl>
+      <emu-alg>
+        1. <ins>If _x_ is a Decimal128 object, then</ins>
+          1. <ins>Let _m_ be ! Call (_x_.[[toString]], « { [[normalize]]: *false* , [[format]]: *"decimal"* , [[roundingMode]]: _unsignedRoundingMode_ } »).</ins>
+          1. <ins>Let _f_ be Decimal128Exponent(_x_).</ins>
+          1. <ins>Let _n_ be Decimal128Significand(_x_).</ins>
+        1. Let _f_ be _maxFraction_.
```

Comment:
The plumbing of the Decimal128 down to here seems right, but I think the code here isn't quite right. For example, `_f_` is defined twice.

Reply:
Quite right. I didn't mean to define a variable twice. I was just trying to bind a value in one if the argument is a Decimal128 object, and a different way if it's a mathematical value. I've fixed that now.